src/platforms/linux/linuxnetworkwatcher.h

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef LINUXNETWORKWATCHER_H
+#define LINUXNETWORKWATCHER_H
+
+#include "networkwatcherimpl.h"
+
+#include <QMap>
+#include <QVariant>
```

Comment:
Question about includes here for my own learning: Why, in this case, do we not do the forward include (? forgotten the exact terminology), that you suggested I use somewhere else?

Reply:
You can use forwarding declaration only in case of pointers and references. In this scenario, I cannot use references or pointers for how dbus slots/signals work.

Reply:
Thank you!